templates/trusted-node/cdk-node-config.toml

```diff
@@ -63,7 +59,6 @@ SanityCheckRPCURL = "http://{{.sequencer_name}}{{.deployment_suffix}}:{{.zkevm_r
                         HTTPHeaders = []
 
 [Aggregator]
-        FinalProofSanityCheckEnabled = false
```

Comment:
I think you recommended that this value was in place when the proofs were failing previously. Why are we removing it now?

Reply:
The code in the aggregator that was performing that check was removed, so the parameter is not needed anymore.